ees_sharepoint/sharepoint_client.py

```diff
@@ -114,12 +114,10 @@ def get_query(start_time, end_time, param_name):
             Returns:
                 query: query for each object"""
         query = ""
-        if param_name in ["sites", "lists"]:
+        if param_name == "sites":
             query = f"?$filter=(LastItemModifiedDate ge datetime'{start_time}') and (LastItemModifiedDate le datetime'{end_time}')"
+        elif param_name == "lists":
+            query = f"?$expand=RootFolder&$filter=(LastItemModifiedDate ge datetime'{start_time}') and (LastItemModifiedDate le datetime'{end_time}') and (Hidden eq false)"
         else:
-            query = f"$filter=(Modified ge datetime'{start_time}') and (Modified le datetime'{end_time}')"
-            if param_name == "list_items":
-                query = "?" + query
-            else:
-                query = "&" + query
```

Comment:
I see you've deleted this part, is this intended?

Reply:
Yes, we no longer need to add an if-else. Earlier, for drive-items we had to use `&` and in list-items we were using `?`. 
Now with the recent change for fetching list-items:

![image](https://user-images.githubusercontent.com/90465691/213138676-c4555a28-3360-4af1-a6d3-fad48f636c0e.png)
we no longer need this logic since the prefix will for both list_items and drive_items will have `&`